packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1995,7 +1995,7 @@ class AnimatedPhysicalModel extends ImplicitlyAnimatedWidget {
   const AnimatedPhysicalModel({
     super.key,
     required this.child,
-    required this.shape,
+    this.shape = BoxShape.rectangle,
```

Comment:
`PhysicalModel()` has a default argument here, so I added one to `AnimatedPhysicalModel()` to match.